src/utils/config.ts

```diff
@@ -43,25 +45,27 @@ export function getConfig(): StorageConfigType {
   dotenv.config()
 
   return {
+    adminApiKeys: getOptionalConfigFromEnv('ADMIN_API_KEYS') || '',
+    adminRequestIdHeader: getOptionalConfigFromEnv('ADMIN_REQUEST_ID_HEADER') || 'request-id',
```

Comment:
I'm leaving it as fastify's default to avoid this being a breaking change for anyone else. It's easy enough for us to override it in our internal environment. Could just leave it undefined if preferred.

Reply:
Good call. As for leaving it undefined, it might make it clear that we're using Fastify's default, but I'm fine either way.